client/ayon_blender/api/pipeline.py

```diff
@@ -530,17 +542,33 @@ def containerise(name: str,
     for obj in nodes:
         container.objects.link(obj)
 
+    if not container.get(AYON_PROPERTY) and (
+        not container.get(AVALON_PROPERTY)
+        ):
+            container[AYON_PROPERTY] = dict()
+
+    ayon_schema = (
+        "openpype:container-2.0"
+        if container.get(AVALON_PROPERTY)
+        else "ayon:container-3.0"
+    )
+    container_id = (
+        AVALON_CONTAINERS
+        if container.get(AVALON_PROPERTY)
+        else AYON_CONTAINERS
+    )
```

Comment:
This logic makes no sense :)... We're creating a NEW collection. The container object will hence NEVER have any of these attributes.

I think we either need to add a `todo` if for now we want to keep the old ids/schemas OR just start using the new ones here directly.